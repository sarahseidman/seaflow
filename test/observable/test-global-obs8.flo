int apple(int x) {
    printi(x);
    return x;
}

int $a = 5;

subscribe(apple, $a);     /* 5           */
subscribe(apple, $a);     /* 5           */
subscribe(apple, $a);     /* 5           */
subscribe(apple, $a);     /* 5           */




$a = 50;                  /* 50 50 50 50 */