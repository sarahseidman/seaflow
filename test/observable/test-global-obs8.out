5
5
5
5
50
50
50
50